### ISSUE TITLE
Problem: deciding (fairly and openly) who the best contributor is

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#621]
Problem: Every 2 weeks, I'm going to award the best contributor $100, it's not meant to be an incentive but a token of appreciation for making my life (and everyone else involved in the project) easier. All contributors to the project are eligible.

These things can sometimes go horribly wrong, so I want to first check that everyone is onboard with the idea itself as well as the criteria for choosing choosing the best contributor. 

I propose that the best contributor will be whoever: 
1. best follows the rules for paid contributors in the Readme file (if you have questions about anything there, please ask)
2. pays attention to detail with their code and verifies that it doesn't break any existing features,
3. does the little housekeeping things that makes life easier for all of us: 
- commenting before starting work on an issue, 
- not reserving multiple issues at once - this is a [blocking process](https://en.wikipedia.org/wiki/Blocking_(computing)) (but it's fine when they are related and need to be worked on at the same time), 
- commenting when an issue has been solved, 
- anything else that makes things flow better.

Everyone participating in the project should vote for the best candidate.

Next award date: Friday March 30th. Voting starts 24 hours before.

I'm hoping to hear from everyone to let me know how they feel about this, whether there's anything that should be changed, or whether this is a silly idea.

ping @anbud @AshHimself @janat08 @MadanBhandari @samuelralak @sjmackenzie

**Comment by samuelralak** [comment]
Its a good idea and am fully onboard. its a good motivation too :). but whats the voting process; how will we cast our votes?

**Comment by gazhayes** [comment]
> but whats the voting process; how will we cast our votes?

I'll make an issue every voting day and people can say who they want to vote for and why. If they want to email me privately it's also ok.

**Comment by AshHimself** [comment]
Great idea! I'm all for it.

**Comment by anbud** [comment]
Yap, I agree, it's a great idea. :)

**Comment by gazhayes** [comment]
As @janat08 and @samuelralak have been away for a while but are expected back shortly, I think it's better to let this run for another week and then vote.

**Comment by MadanBhandari** [comment]
Great idea, As I was been away for a while, but I am back now :+1:  I am in.

**Comment by gazhayes** [comment]
Ah yes, that's a good point. "you voted for that guy last time so I'm not voting for you" type funny business could creep in.

**Comment by samuelralak** [comment]
good point @AshHimself. still deciding on a vote, I have two guys who I think deserve it 😅

**Comment by samuelralak** [comment]
vote cast via email

**Comment by anbud** [comment]
I'll cast my vote via email soon, I agree, 'secret' voting is kinda easier and prevents future biases.

**Comment by janat08** [comment]
voting for gazhayes! I hope he returns the favor.

**Comment by gazhayes** [comment]
Ha! @janat08 you can't vote for me.

**Comment by janat08** [comment]
I vote for myself.

**Comment by janat08** [comment]
did any1 win?
